include/libopencm3/stm32/g4/pwr.h

```diff
@@ -0,0 +1,195 @@
+/** @defgroup pwr_defines PWR Defines
+ *
+ * @ingroup STM32G4xx_defines
+ *
+ * @brief <b>Defined Constants and Types for the STM32G4xx Power Control</b>
+ *
+ * @version 1.0.0
+ *
+ * @author @htmlonly &copy; @endhtmlonly 2016 Benjamin Levine <benjamin@jesco.karoo.co.uk>
+ * @author @htmlonly &copy; @endhtmlonly 2019 Guillaume Revaillot <g.revaillot@gmail.com>
+ * @author @htmlonly &copy; @endhtmlonly 2020 Ben Brewer <ben.brewer@codethink.co.uk>
+ *
+ * @date 29 July 2020
+ *
+ * LGPL License Terms @ref lgpl_license
+ *  */
+
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2016 Benjamin Levine <benjamin@jesco.karoo.co.uk>
+ * Copyright (C) 2020 Ben Brewer <ben.brewer@codethink.co.uk>
```

Comment:
I find it _realllllly_ problematic when people add copyright claims, but can't get their own stories straight.  Is this BenjaminL, Guillaume and BenB, or just Benjamin and Ben? What, if anything, is _actually_ copyright BenjaminL, in 2016, in this file?  based on a file? sure, APIs aren't copyrightable directly, witness google/oracle, so this is.... unclear/misleading, but... whatever, nothing new I guess?

Reply:
None of the files are mine originally, I copied and pasted the closest relevant file and made the minimal changes required to make it STM32G4 compatible, so I thought it made sense to copy the original copyright as I guess legally we both have copyright claims to the file? Happy to change it if there's another process to follow.

Reply:
well, you have a different set of authors in the "Copyright" and in the "Authors" and I don't think that sounds right :)